src/jira/JiraClient.ts

```diff
@@ -21,6 +21,10 @@ export interface IJiraWorklog {
   date: Date;
 }
 
+export interface IJiraAlias {
```

Comment:
I would move that to the `types.ts`-file as it is not a constraint from the api, but our own definition.

Reply:
I extracted the alias management to a AliasStore